src/genie/libs/parser/iosxe/show_ap.py

```diff
@@ -0,0 +1,119 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ====================
+# Schema for:
+#  * 'show ap summary'
+# ====================
+class ShowApSummarySchema(MetaParser):
+    """Schema for show ap summary."""
+
+    schema = {
+        "ap_summary": {
+            "ap_neighbor_count": int,
+            str: {
+                "slots_count": int,
+                "ap_model": str,
+                "ethernet_mac": str,
+                "radio_mac": str,
+                "location": str,
+                "ap_ip_address": str,
+                "state": str
+            }
+        }
+    }
+
+
+# ====================
+# Parser for:
+#  * 'show ap summary'
+# ====================
+class ShowApSummary(ShowApSummarySchema):
+    """Parser for show ap summary"""
+
+    cli_command = ['show ap summary']
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
```

Comment:
This will crash when output is None.
It should be 
out = self.device.execute(self.cli_command[0])

Reply:
@badalsoyantar, I changed cli_command to `cli_command = 'show ap summary'`.